### PR TITLE
added fontawesome to use common icons.

### PR DESCRIPTION
**Pull request by SKOLZ**
Summary: added fontawesome to use it's custom icons, this PR should be rebased after merging the bootstrap-sass one to use the new scss syntax.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -13,6 +13,9 @@ gem 'pg'
 # Use unicorn as the app server
 gem 'unicorn'
 
+# Use fontawesome for common icons
+gem "font-awesome-rails"
```
Comment (epintos) [inline Gemfile]: Single quotes

File: app/views/application/index.html.slim [hunk 2]
```diff
@@ -1,3 +1,4 @@
 .text-center
+  = fa_icon "heart"
```
Comment (epintos) [inline app/views/application/index.html.slim]: This was on purpose?

Comment (SKOLZ) [inline app/views/application/index.html.slim]: yes, this was to test that font awesome was working. do you want me to remove it? or should we leave it?

Comment (epintos) [inline app/views/application/index.html.slim]: Yeah, i would remove it

**Comment by epintos** [comment]
:+1:

**Pull request merged**